Fix: add install requires for pip installation

pahumadad opened the pull request:
Include the requirements packages in the setup file in order to be installed in the notifier pip installation.

These requirements must be updated any time a new requirement package be added to the project.

pahumadad committed:
Add install_requires for pip installation

fortizc commented on setup.py:
```diff
@@ -32,7 +32,14 @@
+        'kombu==4.1.0',
+        'pytz==2017.3',
+        'vine==1.1.4'
+    ],
```
This list comes from the requirements.txt file in this repo? because then the only real dependency here is `Celery` and the others packages are celery's dependencies right? We must allow to Celery manage his own dependencies
In others hand, is there any way to set the version using something like a glob or regexp expression? for example, it will be interesting set `Celery` version like ```celey==4.x.x``` because if our software works with 4.1.0 then it should work with 4.1.1, 4.2.1, etc

pahumadad replied:
I agree with all, only one concern: "... allow to Celery manage his own dependencies..." means that in our setup and requirements.txt we only have to specify "celery==4.x.x", right?

pahumadad committed:
Update to set celery as the only requirement for now

Now we set Celery as the only requirement and let Celery to manage his
own requirements. Also, we set the Celery version with ">=4, <5" which
means we want the latest minor and patch version with for the current
major version, 4.

pahumadad committed:
Now 'install_requires' is reading from the requirements.txt file

pahumadad commented:
@fortizc, you were right. Read the `install_requires` from a file outside the `setup` file is a much better way to solve this!

pahumadad commented:
Now, we use the latest minor and patch release for the current major version of celery...., but are you totally sure that a different minor version couldn't break our code?

fortizc commented:
> Now, we use the latest minor and patch release...

Is mayor.minor.patch

> but are you totally sure that a different minor version couldn't break our code?

Yes, see this [Python versioning system is compatible with semantic versioning](https://www.python.org/dev/peps/pep-0440/#semantic-versioning)

pahumadad commented:
We're right then!
